Makefile

```diff
@@ -78,9 +123,52 @@ client\:dump:
 client\:add-schema:
 	./build/defradb client schema add -f examples/schema/bookauthpub.graphql
 
+.PHONY: deps\:lint-go
+deps\:lint-go:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.54
+
+.PHONY: deps\:lint-yaml
+deps\:lint-yaml:
+ifneq (, $(shell which yamllint))
+	$(info YAML linter 'yamllint' already installed.)
+else ifneq (, $(shell which pip))
+	-pip install --user yamllint
+else
+	$(info YAML linter 'yamllint' and `pip` both not found.)
+endif
+ifeq (, $(shell which yamllint)) # If yamllint still not installed then try this: 
```

Comment:
question:  Sorry I missed this on first review.  Are you sure this is really worth it?  I am somewhat sceptical - did you hit problems in the CI?

Reply:
The first method failed to install on one of the machines (not ci, locally) so provided the alternative. This just ensure like previously when we did `make deps:lint` it can install the linter regardless of which machine we are on. Not as easy to do this since its not a go tool haha like the go linter which would install easily.

Reply:
This only needs to work in the CI though really no?  I'd rather not have us maintain a list of every possible installation pathway for every non-go dependency we end up having.  Not to mention that it will 'break' every time a new pathway becomes available/common.

Can just add a line in the readme noting this linter is a dependency, and then the machine owner can install it in whatever way they prefer.  Having that choice if probably preferred by a bunch of people who may have multiple package managers installed (e.g. if they have one main, and then at some point in the distant past had to install another for a very specific nuisance package, and that one-off gets auto-picked up by us first).

Reply:
I'm also not sure people would appreciate random `sudo` calls in our make file either.

Reply:
Yea, I guess that's one way so we don't have to worry about the nastiness of building and maintaining every possible way to download a dependency. We could perhaps adopt the strategy that if it is not-a-go tool we just document it and not provide utility methods to install it.

Reply:
- [ ] will remove, unless someone else has a stronger opinion

Reply:
I'm also in favour or removing it 👍

Reply:
Removed, please re-review